Sources/Host/Linux/ARM/PTraceARM.cpp

```diff
@@ -136,6 +137,43 @@ ErrorCode PTrace::readCPUState(ProcessThreadId const &ptid, ProcessInfo const &,
   return kSuccess;
 }
 
+uint32_t PTrace::getStoppointData(ProcessThreadId const &ptid) {
+  pid_t pid;
+
+  ErrorCode error = ptidToPid(ptid, pid);
+
+  if (error != kSuccess)
+    return 0;
+
+  //
+  // Retrieve the information about Hardware Breakpoint, if supported.
+  //
+  unsigned int value = 0;
+  if (wrapPtrace(PTRACE_GETHBPREGS, pid, 0, &value) < 0) {
+    DS2LOG(Debug, "hardware breakpoint support disabled, error=%s",
+           Stringify::Errno(errno));
+    return 0;
+  }
+
+  if (((value >> 24) & 0xff) == 0) {
+    return 0;
+  }
+
+  return value;
+}
+
+int PTrace::getMaxHardwareBreakpoints(ProcessThreadId const &ptid) {
+  return std::min<int>(getStoppointData(ptid) & 0xff, ds2::kMaxHWStoppoints);
```

Comment:
I don't think `kMaxHWStoppoints` is defined in this commit.
Also, I'm fairly certain you don't need to specify the template type (`int`) for `std::max` because it will be deduced from arguments, unless you have a type mismatch.